ui/src/utils/jupyterhub.test.ts

```diff
@@ -94,6 +95,43 @@ describe('JupyterHub utils', () => {
     expect(result).toBe('Python');
   });
 
+  test('returns a friendly date string for recent change', () => {
+    const result = getFriendlyDateStr(new Date());
+    expect(result).toBe('Just now');
+  });
+
+  test('returns a friendly date string for change within 1 minute', () => {
+    const currentDate = new Date();
+    const result = getFriendlyDateStr(
+      new Date(currentDate.setMinutes(currentDate.getMinutes() - 1)),
+    );
+    expect(result).toBe('1 minute ago');
+  });
+
+  test('returns a friendly date string for change within minutes', () => {
+    const currentDate = new Date();
+    const result = getFriendlyDateStr(
+      new Date(currentDate.setMinutes(currentDate.getMinutes() - 5)),
+    );
+    expect(result).toBe('5 minutes ago');
+  });
+
+  test('returns a friendly date string for change within hours', () => {
+    const currentDate = new Date();
+    const result = getFriendlyDateStr(
+      new Date(currentDate.setHours(currentDate.getHours() - 5)),
+    );
+    expect(result).toBe('5 hours ago');
+  });
+
+  test('returns a friendly date string for change within days', () => {
+    const currentDate = new Date();
+    const result = getFriendlyDateStr(
+      new Date(currentDate.setHours(currentDate.getHours() - 120)),
+    );
+    expect(result).toBe('5 days ago');
+  });
+
```

Comment:
These are not related, right?

Reply:
Correct. Just taking the time to add some additional unit tests for coverage.

Reply:
Ah okay, thanks for clarifying.